webrtc.html

```diff
@@ -4790,23 +4770,74 @@ <h2>Methods</h2>
+                      <a href="https://heycam.github.io/webidl/#dfn-convert-ecmascript-to-idl-value">converting</a>
+                      the ECMAScript object represented by <var>keygenAlgorithm</var> to an
+                      <code>RTCCertificateExpiration</code> dictionary.</li>
+                    <li>If the conversion fails with an <var>error</var>,
+                      return a promise that is <a>rejected</a> with this <var>error</var>.</li>
+                    <li>If <var>certificateExpiration</var>.<code>expires</code>
+                      is not undefined, set <var>expires</var> to
+                      <var>certificateExpirations</var>.<code>expires</code>.</li>
+                  </ol>
+                </li>
+                <li>Let <var>normalizedKeygenAlgorithm</var> be the result of
+                  <a data-cite="!WebCryptoAPI#dfn-normalize-an-algorithm">normalizing an algorithm</a>
+                  with an operation name of <code>generateKey</code> and a
+                  <a data-cite="!WebCryptoAPI#dfn-supportedAlgorithms">supportedAlgorithms</a>
+                  value specific to production of certificates for
+                  <code>RTCPeerConnection</code>.</li>
+                <li>If the above step produces an <var>error</var>, return a
+                  promise that is <a>rejected</a> with this
+                  <var>error</var>.</li>
+                <li>If the <var>normalizedKeygenAlgorithm</var> parameter
+                  identifies an algorithm that the <a>user agent</a> cannot
+                  or will not use to generate a certificate for
+                  <code>RTCPeerConnection</code>, return a promise that is
+                  <a>rejected</a> with a <code>DOMException</code> of type
+                  <code>NotSupportedError</code>. In particular,
+                  <var>normalizedKeygenAlgorithm</var> MUST be an asymmetric
+                  algorithm that can be used to produce a signature used to
+                  authenticate DTLS connections.</li>
+                <li>Let <var>p</var> be a new promise.
+                <li>Run the following steps in parallel:
+                  <ol>
+                      <li>Perform the generate key operation specified by
+                        <var>normalizedKeygenAlgorithm</var> using
+                        <var>keygenAlgorithm</var>.</li>
+                      <li>Let <var>generatedKeyingMaterial</var> and
+                        <var>generatedKeyCertificate</var> be the private
+                        keying material and certificate generated by the
+                        above step.</li>
+                      <li>Let <var>certificate</var> be a new
+                        <code><a>RTCCertificate</a></code> object.</li>
+                      <li>Set <var>certificate</var>.[[\Expires]] to the
+                        current time plus <var>expires</var> value. A <a>user
```

Comment:
Should "current time" be more precise? In other places we say `<a>performance.timeOrigin</a> + <a>performance.now()</a>`.

Reply:
For now, I kept current time with is a DOMTimeStamp.
In other places, we are using DOMHighResTimeStamp while we are using DOMTimeStamp here.